src/main/resources/org/jenkinsci/plugins/electricflow/ElectricFlowDeployApplication/config.jelly

```diff
@@ -117,20 +117,40 @@ All rights reserved.
                     }
 
 
-                    var row1 = table.insertRow(table.rows.length - 2);
-                    row1.className = "_ef_da_row";
-                    if (parameters.length) {
-                        row1.appendChild(createTitleForDeployApplication("Deploy Parameters"));
-                        parameters.forEach(function (elem) {
-                            var row = table.insertRow(table.rows.length - 2);
-                            row.className = "_ef_da_parameters _ef_da_row";
-                            addParameterRowForDeployApplication(row, elem.actualParameterName, elem.value, false);
-                        });
+                    if (table == null) {
+                        if (parameters.length) {
+                            var el = document.createElement("div");
+                            el.innerHTML = "&lt;br&gt;&lt;b&gt;Deploy Parameters&lt;/b&gt;";
+                            el.className = "_ef_da_row";
+                            var div = getElementByXpath("//*[@id=\"ef_da_environmentName\"]");
```

Comment:
I'd suggest replacing this with `document.getElementById="ef_da_environmentName"`. `getElementByXPath` is not native JavaScript.

Reply:
thanks, updated accordingly. PS. we still use getElementByXpath in some places

Reply:
thanks, updated accordingly. PS. we still use getElementByXpath in some places